cycode/cyclient/scan_client.py

```diff
@@ -108,11 +118,16 @@ def multiple_zipped_file_scan_async(
         )
         return models.ScanInitializationResponseSchema().load(response.json())
 
-    def get_scan_details_path(self, scan_type: str, scan_id: str) -> str:
-        return f'{self.get_scan_service_url_path(scan_type)}/{scan_id}'
+    def get_scan_details_path(self, scan_type: str, scan_id: str, should_use_scan_service: bool = False) -> str:
```

Comment:
it looks that `get_scan_details_path` is used only in `get_scan_details` which is used in `poll_scan_results`. and the polling is async scan (via scan service only). no need of `should_use_scan_service` here